docker/package/tezos_setup_wizard.py

```diff
@@ -424,12 +424,14 @@ def import_baker_key(self):
 
     def register_baker(self):
         print()
-        tezos_client_options = self.get_tezos_client_options()
-        baker_alias = self.config["baker_alias"]
-        proc_call(
-            f"sudo -u tezos {suppress_warning_text} octez-client {tezos_client_options} "
-            f"register key {baker_alias} as delegate"
-        )
+        # TODO uncomment after https://gitlab.com/tezos/tezos/-/issues/4200 resolved
+        # and validation fixed
+        # tezos_client_options = self.get_tezos_client_options()
+        # baker_alias = self.config["baker_alias"]
+        # proc_call(
+        #    f"sudo -u tezos {suppress_warning_text} octez-client {tezos_client_options} "
+        #    f"register key {baker_alias} as delegate"
+        # )
```

Comment:
Actually, now that we are using this logic (of running it without `--dry-run`) perhaps it makes sense to make this solution permanent.

AFAIK one needs an address that is a delegate both for baking and voting (note that both wizards call `import_key` from the wizard structure).

So IMO the right solution is to call the "register delegate" command (without `--dry-run`) and until it has an exit code of `0` (either because a delegate got registered or because it was already) in `import_key`.

@krendelhoff2 how does that sound to you?
If you implement this solution please make sure to test both wizards, because I haven't checked that it works alright.

Reply:
Actually yes, it sounds perfectly fine. Reregistering delegate is safe and free, so it does not cause any problem anywhere.

Reply:
I'll reword commits before merge